.github/workflows/coverity-scan.yml

```diff
@@ -0,0 +1,88 @@
+name: Coverity Scan
+on:
+  schedule:
+    - cron: '0 0 1 * *' # Monthly (1st day of month at midnight)
+
+  workflow_dispatch: # Mainly for testing. Don't forget the Coverity usage limits.
+
+env:
+  LIBTORRENT_VERSION_TAG: v1.2.14
+
+jobs:
+
+  coverity_scan:
+    name: Scan
+
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        qbt_gui: ["GUI=ON"]
+      fail-fast: false
+
+    runs-on: ${{ matrix.os }}
+
+    defaults:
+      run:
+        shell: bash
```

Comment:
If you want, you could simplify/omit a lot of these declarations:
* use `runs-on: ubuntu-20.04`
* inline `qbt_gui` variable
* remove `defaults:` the default shell on ubuntu is already bash IIRC.
* `fail-fast:` is redundant since there are only 1 job.
* Therefore `strategy:` could be removed.

Reply:
Done.